_schemaV2.graphql

```diff
@@ -17298,7 +17319,8 @@ type Viewer {
     # The slug or ID of the Partner
     partnerID: String!
     size: Int
-  ): PartnerArtistDocumentConnection
+  ): PartnerArtistDocumentsConnectionConnection
```

Comment:
there is a double `Connection` here I wonder if we can update the types so it is only one?

Reply:
Updated with https://github.com/artsy/metaphysics/pull/4489/commits/c4ecc372fa614158a3cf9b1f9705c2b249c67b11!